doc/nrf/ug_nrf5340.rst

```diff
@@ -1,3 +1,5 @@
+.. |multi_image| replace:: These samples are built for the application core and include the network core application as child image in a multi-image build (see :ref:`ug_nrf5340_multi_image`).
```

Comment:
```suggestion
.. |multi_image| replace:: These samples are built for the application core and may include the network core application as child image in a multi-image build (see :ref:`ug_nrf5340_multi_image`).
```

Reply:
You need explicit support in `samples/CMakeLists.txt` (or somewhere else in the build system) for a sample to be automatically added as a child image.